cmake/Modules/FindTileDB_EP.cmake

```diff
@@ -48,30 +48,30 @@ if (NOT TILEDB_FOUND)
     # Try to download prebuilt artifacts unless the user specifies to build from source
     if(DOWNLOAD_TILEDB_PREBUILT)
         if (WIN32) # Windows
-          SET(DOWNLOAD_URL "https://github.com/TileDB-Inc/TileDB/releases/download/2.20.1/tiledb-windows-x86_64-2.20.1-249c024.zip")
-          SET(DOWNLOAD_SHA1 "988fdb6f9cf26ae4b9c06951dda6abfb288f51d8")
+          SET(DOWNLOAD_URL "https://github.com/TileDB-Inc/TileDB/releases/download/2.21.0/tiledb-windows-x86_64-2.21.0-0ea9c13.zip")
```

Comment:
Just FYI, we have a new https://github.com/TileDB-Inc/TileDB/releases/download/2.21.0/DownloadPrebuiltTileDB.cmake helper and associated csv file list, in case you want to try that (one file, no more calculating all the sha256 manually). Ask @dudoslav if any questions.

Reply:
that's great thanks, I can use it to automate this procedure.